### ISSUE TITLE
Missing error/warning markers in AADL navigator view 

### ISSUE THREAD

**Issue by lwrage** [osate/osate2#2609]
**Summary**
The AADL Navigator no longer shows error markers on elements in expanded AADL model files.

This used to work in previous versions.

**Steps To Reproduce**

1. Paste model below into AADL editor and save it as _P.aadl_.
2. Expand _P.aadl_ file in AADL navigator
3. The file contains error markers on flow implementations for `fl`
4. No error markers on flows in the AADL navigator

```aadl
package P
public

	thread T
		features
			i: in data port;
			o: out data port;
		flows
			fl: flow path i -> o;
	end T;

	thread implementation T.i
		calls
			seq: {
				cl: subprogram SP;
			};
		connections
			pc1: parameter i -> cl.pi;
			pc2: parameter cl.po -> o;
		flows
			-- cannot specify flow through called subprogram
			-- both of the following are not allowed
			fl: flow path i -> pc1 -> cl -> pc2 -> o;
			fl: flow path i -> pc1 -> cl.fp -> pc2 -> o;
	end T.i;

	subprogram SP
		features
			pi: in parameter;
			po: out parameter;
		flows
			fp: flow path pi -> po;
	end SP;
end P;
```

**Screenshots**
OSATE 2.9.0:
![image](https://user-images.githubusercontent.com/750311/110156411-ea18a100-7db4-11eb-858c-5f6fbebd564e.png)

OSATE 2.9.1:
![image](https://user-images.githubusercontent.com/750311/110156583-264c0180-7db5-11eb-8417-dacdc7204fa3.png)

**Desktop (please complete the following information):**
 - OSATE Version: 2.9.1
 - Operating System: Linux, Windows

**Comment by lwrage** [comment]
Looks like this is caused by the fix to #2430.

**Comment by AaronGreenhouse** [comment]
Problem wasn't the imaging caching.  I thought maybe I wasn't including the decorations correctly, but I was.  The problem was that the original `EObject` represented by the wrapper was coming from a different `ResourceSetImpl` than the was being used when looking at the markers of any children.  That meant the `EcoreUtil.isAncestor()` predicate was always returning false.  I changed the factory to be able to share the resource set used by the caller in `AadlElementContentProvider.getChildren()`.
